StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsSessionManifest.swift

```diff
@@ -78,4 +84,5 @@ struct FinancialConnectionsSessionManifest: Decodable {
     let experimentAssignments: [String: String]?
     let assignmentEventId: String?
     let skipSuccessPane: Bool?
+    let manualEntryMode: ManualEntryMode
```

Comment:
backend model to double-check:
- https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/api/resource/link_account_session_manifest_resource.rb#L22-25